src/styles/blocks/icons.scss

```diff
@@ -0,0 +1,17 @@
+.icon {
+  display: block;
+  height: 24px;
+  width: 24px;
+}
+
+.icons__icon--phone {
+  background-image: url(../images/icon-phone.png);
+}
+
+.icons__icon--menu {
+  background-image: url(../images/icon-burger-menu.png);
```

Comment:
Menu Icon should change on hover:
![image](https://github.com/mate-academy/layout_landing-page/assets/52002856/e9b13f47-0cff-4145-a214-7303356cad45)

Reply:
Done